torch_geometric/utils/scatter.py

```diff
@@ -1,66 +1,91 @@
-import warnings
 from typing import Optional
 
 import torch
-import torch_scatter
 from torch import Tensor
 
-import torch_geometric
-
 major, minor, _ = torch.__version__.split('.', maxsplit=2)
 major, minor = int(major), int(minor)
 has_pytorch112 = major > 1 or (major == 1 and minor >= 12)
 
+if has_pytorch112:  # pragma: no cover
```

Comment:
Can we also add a test for https://github.com/pyg-team/pytorch_geometric/issues/5352 to see if it works as expected now?

Reply:
Done.